cmd/example-app/main.go

```diff
@@ -187,10 +221,79 @@ func main() {
 		)
 	}
 	{
-		server := &http.Server{Addr: *addr}
+		var tlsConfig *tls.Config = nil
+		if flag.Lookup("self-signed") != nil || *tlsCert != "" {
```

Comment:
```suggestion
		if selfSigned != nil || *tlsCert != "" {
```

Why not variable?

Reply:
Marking resolved. This was split out into #570. Thanks!